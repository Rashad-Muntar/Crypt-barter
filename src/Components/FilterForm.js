import React from 'react';
import PropTypes from 'prop-types';
import '../Assets/navbar.css';

const FilterForm = ({ onCrytoFilter }) => (
  <div className="search">
    <i className="search icon" />
    <input type="text" placeholder="Enter crypto name" onChange={onCrytoFilter} />
  </div>

);

FilterForm.propTypes = {
  onCrytoFilter: PropTypes.func,
};

FilterForm.defaultProps = {
  onCrytoFilter: () => {

  },
};

export default FilterForm;
