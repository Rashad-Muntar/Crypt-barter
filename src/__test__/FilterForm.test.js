import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import FilterForm from '../Components/FilterForm';
// import Crypto from '../Components/Crypto'

describe('Home', () => {
  beforeEach(() => {
    render(<FilterForm />);
  });

  it('Renders Filter form component', () => {
    const tree = renderer.create(<FilterForm />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('Renders Filter form component', () => {
  //   const tree = renderer.create(< >).toJSON();
  //   expect(tree).not.toMatchSnapshot();
  // });
});
