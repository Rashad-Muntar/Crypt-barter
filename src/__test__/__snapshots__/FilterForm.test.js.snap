// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Renders Filter form component 1`] = `
<div
  className="search"
>
  <i
    className="search icon"
  />
  <input
    onChange={[Function]}
    placeholder="Enter crypto name"
    type="text"
  />
</div>
`;
